### PR TITLE
Sidebar responsiveness fix

### PR DESCRIPTION
**Pull request by andreo141**
## ID
[CLBV-216](https://binnenland.atlassian.net/browse/CLBV-216)

 ## Description

Sidebar was not responsive when displayed on smaller screens. 

 ## Type of change

 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Other

 ## How to test

Resize the window to test the responsiveness of the sidebar container and the breadcrumb trail

### CODE REVIEW

File: app/templates/core-data.hbs
Comment (lagartoverde) [inline app/templates/core-data.hbs:1]: You can extract this to a component, and then you are not repeating the same code in both pages

**Review by lagartoverde — CHANGES_REQUESTED** [review]
Make a component is better, then we can add it to new pages and we are not repeating code

**Review by lagartoverde — CHANGES_REQUESTED** [review]
image
Now my login looks like this xd
image
Also on this screen we have a new scroll bar that we don't need
image
And still the responsiveness of the menu is not ideal

**Comment by andreo141** [comment]
I will make a new PR with the new login page. Will close this one.

**Pull request closed without merging**